src/sets.jl

```diff
@@ -124,31 +149,41 @@ function distance_to_set(::DefaultDistance, v::AbstractVector{<:Real}, s::MOI.Po
     return max(result, zero(result))
 end
 
-function distance_to_set(::DefaultDistance, vs::AbstractVector{<:Real}, s::MOI.DualPowerCone)
+function distance_to_set(::NormedEpigraphDistance{p}, vs::AbstractVector{<:Real}, s::MOI.DualPowerCone) where {p}
     _check_dimension(vs, s)
     u = vs[1]
     v = vs[2]
     w = vs[3]
     e = s.exponent
     ce = 1-e
     result = abs(w) - (u/e)^e * (v/ce)^ce
-    return LinearAlgebra.norm2(
-        (max(-u, zero(result)), max(-v, zero(result)), max(result, zero(result)))
+    return LinearAlgebra.norm(
+        (max(-u, zero(result)), max(-v, zero(result)), max(result, zero(result))), p,
     )
 end
 
-function distance_to_set(::DefaultDistance, v::AbstractVector{<:Real}, set::MOI.RelativeEntropyCone)
+function distance_to_set(::DefaultDistance, vs::AbstractVector{<:Real}, s::MOI.DualPowerCone)
+    return distance_to_set(NormedEpigraphDistance{2}(), vs, s)
+end
+
+function distance_to_set(::NormedEpigraphDistance{p}, v::AbstractVector{<:Real}, set::MOI.RelativeEntropyCone) where {p}
     _check_dimension(v, s)
     n = (dimension(set)-1) ÷ 2
     u = v[1]
     v = v[2:(n+1)]
     w = v[(n+2):end]
     s = sum(w[i] * log(w[i]/v[i]) for i in eachindex(w))
     result = s - u
-    return LinearAlgebra.norm2(push!(
-        max.(v[2:end], zero(result)),
-        max(result, zero(result)),
-    ))
+    return LinearAlgebra.norm(
+        push!(
+            max.(v[2:end], zero(result)),
+            max(result, zero(result)),
+        ), p,
```

Comment:
Is the last comma expected?

Reply:
yes, I try to leave trailing commas to be able to add a new line without changing the existing lines with arguments, maybe we could have the `p,` on a new line?